microsoft/testsuites/core/azure_image_standard.py

```diff
@@ -894,6 +900,47 @@ def verify_boot_error_fail_warnings(self, node: Node) -> None:
             f" {node.os.name} {node.os.information.version}"
         ).is_empty()
 
+    @TestCaseMetadata(
+        description="""
+        This test will check ERROR, WARNING messages from /var/log/cloud-init.log
+        and also check cloud-init exit status.
+
+        Steps:
+        1. Get ERROR, WARNING messages from /var/log/cloud-init.log.
+        2. If any unexpected ERROR, WARNING messages or non-zero cloud-init status 
+        fail the case.
+        """,
+        priority=1,
+        requirement=simple_requirement(supported_platform_type=[AZURE, READY]),
+    )
+    def verify_cloud_init_error_status(self, node: Node) -> None:
+        cat = node.tools[Cat]
+        if isinstance(self.node.os, CBLMariner):
```

Comment:
Can it apply to all images?

Reply:
The test is meant to for Mariner only to have early regression for our cloud-init development in Mariner but I believe it also applies other distros if you to have this for all distros tested in Lisa .

Reply:
I've removed the line limits the test to Mariner only and applied to all distros